Formula/z/zrok.rb

```diff
@@ -5,6 +5,7 @@ class Zrok < Formula
   sha256 "2980581c45514240598135deed8a999bc65359527ded31a5bb855d05e70f2254"
   # The main license is Apache-2.0. ACKNOWLEDGEMENTS.md lists licenses for parts of code
   license all_of: ["Apache-2.0", "BSD-3-Clause", "MIT"]
+  revision 1
```

Comment:
```suggestion

```

Reply:
The version change is fine, but we should not revision bump this. We don't want to force every user to reinstall. They'll get the changes in the next version bump.

Reply:
I think we must force an upgrade because revision 0 (implied) will not function at all due to the failing client version check lacking the leading v character.

Reply:
That's correct. Revision 0 introduced zrok v1, an upgrade from v0 which had a client-side version check requiring the server's reported version to match a pattern. In zrok v1, the client submits a version string to the server, and the official release requires that version to have prefix `v1`.